gulpfile.js

```diff
@@ -93,6 +98,39 @@ gulp.task('compile_test', function() {
     .pipe(buildTest())
 });
 
+/**
+ * Task only used to verify TypeScript autogenerated typings.
+ */
+gulp.task('compile_autogen_typing', function() {
+  return gulp.src(paths.src)
+    // Compile Typescript into .js and .d.ts files
+    .pipe(buildAutogenTyping())
+
+    // Exclude typings that are unintended (only RTDB and FirebaseApp are
+    // expected to produce typings).
+    .pipe(filter([
+      'lib/src/**/*.js',
```

Comment:
If you pass `rootDir` option to the compiler `lib/src` can be shortened to `lib/`